apps/next/app/server/clerk/index.ts

```diff
@@ -31,11 +33,14 @@ export async function findCreateClerkOrganizationCreatedByUser({
   name: string;
   createdByClerkId: string;
 }): Promise<ClerkOrganization> {
-  const orgs = await clerkClient.users.getOrganizationMembershipList({
+  const { data: orgs } = await clerkClient.users.getOrganizationMembershipList({
```

Comment:
Wrong. Clerk clearly states that `orgs`'s type is an array, so it can't be null. If `orgs` is empty, then the `for` loop simply doesn't execute and a new Clerk organization is created for the user.